de.metas.adempiere.adempiere/client/src/main/java/de/metas/process/ui/ProcessPanel.java

```diff
@@ -605,8 +605,15 @@ public void unlockUI(final ProcessInfo pi)
 			{
 				closeWindow = true;
 			}
+			
+			// #863
+			// Do not show result window for any process that doesn't allow rerun
+			if(!_allowProcessReRun)
```

Comment:
Hi. Yes, it is respected. I tested for invoice candidates that cannot be invoiced ans shipmentschedules that are already shipped. The error messages are still shown in the popup, without the misleading question about restarting.